shuup/front/apps/auth/forms.py

```diff
@@ -54,6 +54,29 @@ def clean_username(self):
 
         return username
 
+    def clean(self):
+        username = self.cleaned_data.get("username")
+        password = self.cleaned_data.get("password")
+
+        if username is not None and password:
+            self.user_cache = authenticate(
+                self.request, username=username, password=password
+            )
+            if self.user_cache is None:
+                try:
+                    user_temp = get_user_model().objects.get(username=username)
+                except ObjectDoesNotExist:
+                    user_temp = None
+
+                if user_temp is not None:
+                    self.confirm_login_allowed(user_temp)
```

Comment:
This also checks whether the contact belongs to the correct shops so should likely be called pretty much always. Now it seems that `self.user_cache is None` when user is inactive and these custom checks are skipped.